src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -100,11 +100,64 @@ static std::string          subscribeContextAvailabilityCollectionName;
 static Notifier*            notifier;
 static bool                 multitenant;
 
+thread_local std::vector<ContextElementResponse*>  savedCerVector;
+
+
+
+/* ****************************************************************************
+*
+* WORKAROUND_2994 - see github issue 2994
```

Comment:
```
...see github issue #2994
```

Typically we use the hash (#) whenever refer to a PR number or issue in comments.

Reply:
Fixed in dcf5197